modules/accounts/src/lib.rs

```diff
@@ -33,19 +40,27 @@ type PalletBalanceOf<T> =
 type NegativeImbalanceOf<T> = <<T as pallet_transaction_payment::Trait>::Currency as Currency<
 	<T as system::Trait>::AccountId,
 >>::NegativeImbalance;
-type DepositBalanceOf<T> = <<T as Trait>::DepositCurrency as Currency<<T as system::Trait>::AccountId>>::Balance;
 
 pub trait Trait: system::Trait + pallet_transaction_payment::Trait + orml_currencies::Trait {
 	type FreeTransferCount: Get<u8>;
 	type FreeTransferPeriod: Get<MomentOf<Self>>;
-	type FreeTransferDeposit: Get<DepositBalanceOf<Self>>;
+	type FreeTransferDeposit: Get<Balance>;
+	type AllCurrencyIds: Get<Vec<CurrencyId>>;
```

Comment:
`AllOtherCurrencyIds`? this shouldn't include the native one right?

Reply:
all non-native currencies, not include native